### DISCUSSION TITLE
Chipped corner for frames importet from Visio

### DISCUSSION THREAD

**Discussion by TheNinth7** [jgraph/drawio#4016] [Q&A]
Hi,

I imported a Visio file into drawio, and many of the boxes have a chipped lower-left corner, like this:

![Screenshot - 03_12_2023 , 08_03_39](https://github.com/jgraph/drawio/assets/118298758/ef7ce4b3-6b74-41ca-b4a6-b1ac66d819cb)

How can I fix this?

The style of the box looks like this:

`verticalAlign=bottom;align=left;overflow=visible;vsdxID=40;fillColor=none;gradientColor=none;shape=stencil(nZBLDoAgDERP0z3SIyjew0SURgSD+Lu9kMZoXLhwN9O+tukAlrNpJg1SzDH4QW/URgNYgZTkjA4UkwJUgGXng+6DX1zLfmoymdXo17xh5zmRJ6Q42BWCfc2oJfdAr+Yv+AP9Cb7OJ3H/2JG1HNGz/84klThPVCc=);strokeColor=#4672c4;strokeWidth=6;spacingTop=8;spacingBottom=8;spacingLeft=8;spacingRight=8;points=[[0.5,0,0]];labelBackgroundColor=none;rounded=0;html=1;whiteSpace=wrap;fontSize=17;fontFamily=Calibri;`

**Comment by davidjgraph** [comment]
No input file provided.

**Reply by TheNinth7** [reply]
Here is the input file. Added the .txt suffix since .drawio was not allowed for upload here:

[evcc-innards.drawio.txt](https://github.com/jgraph/drawio/files/13539979/evcc-innards.drawio.txt)

**Comment by TheNinth7** [comment]
Is it possible to reopen this discussion? Why was it closed?

**Comment by TheNinth7** [comment]
Interestingly this issue occurs when editing the diagrams, when exporting them to PNG but not when embedding them in Docusaurus with this plugin: https://github.com/xiguaxigua/docusaurus-plugin-drawio

Editor:
![Screenshot - 11_12_2023 , 08_32_25](https://github.com/jgraph/drawio/assets/118298758/763e29ba-6fec-4b52-9341-61dc308e432d)

PNG:
![Screenshot - 11_12_2023 , 08_32_43](https://github.com/jgraph/drawio/assets/118298758/00ae866e-bb62-4fc4-a871-5c4668144014)

Docusaurus-Plugin:
![Screenshot - 11_12_2023 , 08_33_15](https://github.com/jgraph/drawio/assets/118298758/02de3f35-68be-4a4b-9b93-0921f960791e)
